packages/react/src/components/task-form/TaskFormDetails.tsx

```diff
@@ -18,146 +19,134 @@ import { PRIORITY_ITEMS, STATUS_ITEMS } from '../../shared/constants';
 
 import { Task } from '../../shared/types/task';
 
-const validationGroup = 'taskFormValidationGroup';
-
 const renderPriorityField = (data: string) => (
-  <>
-    <PriorityTask text={data}></PriorityTask>
-    <TextBox readOnly></TextBox>
-  </>
+  <div className='priority-task-editor-field'>
+    <PriorityTask text={data} showText={false}></PriorityTask>
+    <TextBox
+      className={`priority-task priority-${data.toLowerCase()}`}
+      inputAttr={{ class: 'priority-editor-input' }}
+      readOnly
+      text={data}
+      hoverStateEnabled={false}
+    />
+  </div>
 );
 
 const renderStatusField = (data: string) => (
-  <>
-    <StatusTask text={data}></StatusTask>
-    <TextBox readOnly></TextBox>
-  </>
+  <div className='status-task-editor-field'>
+    <StatusTask text={data} showText={false}></StatusTask>
```

Comment:
Do we need to use status task here? Looks like we can use only text-box

Reply:
status task removed